core/iwasm/libraries/debug-engine/handler.c

```diff
@@ -0,0 +1,598 @@
+/*
+ * Copyright (C) 2021 Ant Group.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#include <bh_log.h>
+#include <handler.h>
+#include <limits.h>
+#include <string.h>
+#include <unistd.h>
+
+#include "debug_engine.h"
+#include "packets.h"
+#include "utils.h"
+#include "wasm_runtime.h"
+
+#define MAX_PACKET_SIZE (0x20000)
```

Comment:
how did you calculate this size?
does this actually need this big buffer?
asking because i want to use this stuff on smaller machines.